lib/test_helper/metrix_fake.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ElixirTools.TestHelper.MetrixFake do
```

Comment:
tests are missing

Reply:
Wait, we test fakes?

Reply:
yep, why not

Reply:
Well we could also test tests but it would be a bit of a waste of time, right? :D